src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export function Person({ person: { name, age, sex, isMarried, partnerName } }) {
+  const spouse = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
```

Comment:
In fact I said it wrong. You should check for `!!age` as if `age === 0`, it will output `0` to the template.